clang/docs/ReleaseNotes.rst

```diff
@@ -977,6 +977,10 @@ Arm and AArch64 Support
   in leaf functions after enabling ``-fno-omit-frame-pointer``, you can do so by adding
   the ``-momit-leaf-frame-pointer`` option.
 
+- Support has been added for the following processors (-mcpu identifiers in parenthesis):
```

Comment:
It should be added that this is for AArch64 target.

Reply:
I've added in this commit 82164cd56dcd6a66a314124067c265d6293247ef.